Main.java

```diff
@@ -9,7 +10,31 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // calculate how much it would cost to purchase different quantities of computer parts
+   String name;
+   System.out.println("Welcome! Enter which computer parts you would like to purchase along with a quantity:");
+   System.out.println("How many Chromebook Chargers would you like?:");
+    // Creates a Scanner used for input
+    Scanner input = new Scanner(System.in);
+     int chromebookCharger = input.nextInt();
+
+
+    System.out.println("How many Replacement Motherboards would you like?:");   
+     int replacementMotherboard = input.nextInt();
+    
+    System.out.println("How many Computer Mouses would you like?:");
+     int computerMouse = input.nextInt();
+
+     double Subtotal = (chromebookCharger*34.99 + replacementMotherboard*127.50 + computerMouse*18.00);
+
+     double Tax = (1.13);
+
+     // declare and calculate the quotient
+   System.out.println(Subtotal*1.13);
+
```

Comment:
Make sure that you are commenting your code throughout. You go through a hug block here without a comment. Also the comment on line 32 is incorrect. Remember that variable names should always begin with a lowercase letter. Subtotal should be subtotal, Tax should be tax, etc. Also, you are missing the calculation for the amount of tax (subtotal*0.13). Finally, watch the output. Try to make it match as closely to the sample as you can.